dev/start.cljs

```diff
@@ -1,14 +1,16 @@
 (ns dat.sys.start
-  (:require [figwheel.client :as fw]
-            [dat.sys.client.app :as app]
-            [dat.view]))
+  (:require [dat.sys.client.app :as app]
+            [dat.view :as view]
+            ))
 
 (enable-console-print!)
 
-(fw/watch-and-reload
- :websocket-url "ws://localhost:3448/figwheel-ws"
- :jsload-callback #(do (app/main)
-                       (dat.view/dispatch! (:app app/system) [:figwheel/reload nil])))
+(defn on-js-reload []
+  (prn "------ Figwheel Has Reloaded ------")
```

Comment:
Would be good to switch to `timber.core/info`, for consistency with the rest of the app.